vs

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
+# This is a small wrapper that lets VegaStrike - Upon the Coldest Sea
+# run from seemingly anywhere
 set -e
 
 VEGASTRIKE_LOCAL_SHARE_DIR="/usr/local/share/vegastrike"
 VEGASTRIKE_SHARE_DIR="/usr/share/vegastrike"
 
-# This is a small wrapper that lets VegaStrike - Upon the Coldest Sea
-# run from seemingly anywhere
+#Let's keep a log of the console
+touch ~/.vegastrike/console.log
```

Comment:
Why create a file and then move it? shouldn't it be other other way around?

Reply:
The idea here is to move the previously captured log out of the way, so it does not get clobbered instantly. 
mv cannot fail graciously if the file it wants to move does not exist. 
There are a whole host of reasons for this file to not exist. 

If the source file for mv does not exist, it fails and the script exits. 
The main purpose of touch is to update the "modified" time of an existing file. Creating the file if it does not exist is a happy side-effect. So, now if the file does not exist, "touch" makes it, otherwise it harmlessly updates it's modified time, allowing the mv command to not fail, as it is guaranteed to have a file to move.

I could have implemented similar logic with:
if -e ~/vegastrike/console.log; then ...
    mv foo bar
fi

However, I nearly always get my if tests wrong in bash, and then it's a bit of back and forth until it works right.

Reply:
let's have it with an if command
```
if [ -f ~/vegastrike/console.log ];
then
    mv foo bar
fi